core/common/token-core/src/test/java/org/eclipse/edc/jwt/rules/ExpirationIssuedAtValidationRuleTest.java

```diff
@@ -60,15 +62,19 @@ void validationKoBecauseExpirationTimeNotRespected() {
                 .contains("Token has expired (exp)");
     }
 
-    @Test
-    void validationKoBecauseExpirationTimeNotProvided() {
+    @ParameterizedTest(name = "Allow exp null: {0}")
+    @ValueSource(booleans = { true, false })
+    void validationKoBecauseExpirationTimeNotProvided(boolean allowNull) {
+        var r = new ExpirationIssuedAtValidationRule(clock, 0, allowNull);
         var token = ClaimToken.Builder.newInstance().build();
 
-        var result = rule.checkRule(token, emptyMap());
+        var result = r.checkRule(token, emptyMap());
 
-        assertThat(result.succeeded()).isFalse();
-        assertThat(result.getFailureMessages()).hasSize(1)
-                .contains("Required expiration time (exp) claim is missing in token");
+        assertThat(result.succeeded()).isEqualTo(allowNull);
+        if (!allowNull) {
```

Comment:
conditionals in test are generally not good, given that this parameterized test has only two inputs, I'd just split in two tests